sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1999,40 +1996,34 @@ public void reassignTasksOfCase(Case caze, boolean forceReassignment) {
 
 	}
 
-	private float calculateCompleteness(Case caze) {
+	@Override
+	public void updateCompletenessTask() {
+		List<String> getCompletenessCheckCaseList = getCompletenessCheckNeededCaseList();
+
+		if (!getCompletenessCheckCaseList.isEmpty()) {
+			long timeStart = System.currentTimeMillis();
+			IterableHelper
+				.executeBatched(getCompletenessCheckCaseList, 100, caseCompletionBatch -> caseService.updateCompleteness(caseCompletionBatch));
+			long timeStop = System.currentTimeMillis();
+			logger.debug(
```

Comment:
Move the logging into the Job method in `CronService`, use varargs for parameters (more efficient).

Reply:
moved